src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -194,7 +194,7 @@ public static class Captions
       public static readonly string EditManually = "Edit manually";
       public static readonly string ShouldColorGroupObservedData = "Color group observed data from same folder when dropping to chart";
       public static readonly string ColorGroupObservedDataContextMenu = "Color group when adding to chart";
-      public static readonly string EditAllCurvesProperties = "Edit options for selected";
+      public static readonly string EditAllCurvesProperties = "Edit Options for Selected";
```

Comment:
This was not following the other context menu. Pascal case expcet for binding words such as to, at, for etc..